.circleci/config.yml

```diff
@@ -332,6 +332,10 @@ jobs:
           name: Refresh configuration as our db will contain active prod config after sync operation
           command: |
             /home/circleci/.platformsh/bin/platform environment:drush cim -y -p $PLATFORM_PROJECT_ID -e $EDGE_BUILD_BRANCH
+      - run:
+          name: Run structure sync import all
+          command: |
+            /home/circleci/.platformsh/bin/platform environment:drush ia --choice=safe -y -p $PLATFORM_PROJECT_ID -e $EDGE_BUILD_BRANCH      
```

Comment:
I'd recommend wrapping the drush command in double quotes so it doesn't confuse the wider platform command parameters